src/index.js

```diff
@@ -69,6 +69,28 @@ const randomCirclePoint = (centerPoint, radius, randomFn = Math.random) => {
   );
 };
 
+/**
+ *
+ * Given a center point C and a smaller radius and bigger radius, return a point that is
+ *  between the two circles
+ */
+const randomPointBetweenCircles = (
+  centerPoint,
+  innerRadius,
+  outRadius,
+  randomFn = Math.random
+) => {
+  if (innerRadius > outRadius) {
+    throw new Error('innerRadus should be smaller');
+  }
+
+  return randomCircumferencePoint(
+    centerPoint,
+    Math.floor(Math.random() * (outRadius - innerRadius + 1) + innerRadius),
```

Comment:
While this seems correct, we need to verify that it results in points that are uniformly distributed on the annulus. I'll do the calculation later today to verify :)